src/core/dbus.c

```diff
@@ -860,6 +860,10 @@ int bus_init_api(Manager *m) {
         if (r < 0)
                 return log_error_errno(r, "Failed to set up API bus: %m");
 
+        r = bus_track_coldplug(bus, &m->subscribed, /* recursive= */ false, m->deserialized_subscribed);
+        if (r < 0)
+                return r;
+        m->deserialized_subscribed = strv_free(m->deserialized_subscribed);
```

Comment:
i'd really rename the field, now that we not just using it during deserialization, but also during serialization. i.e. maybe rename the field to `subscribed_as_strv` or so?

Reply:
ah, the 2nd patch is doing that.

Reply:
I did change it to "subscribed_serialization" as suggested by https://github.com/systemd/systemd/pull/35406#discussion_r1880525310, but in a following commit to separate the logical change from the cosmetic one. I can squash them if you'd like. `subscribed_strv` sounds better to me now though.

Reply:
I made it `subscribed_as_srtv` now.